src/main/java/edu/cornell/kfs/pdp/businessobject/inquiry/CuPaymentDetailInquirable.java

```diff
@@ -0,0 +1,45 @@
+package edu.cornell.kfs.pdp.businessobject.inquiry;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.kns.web.ui.Field;
+import org.kuali.kfs.kns.web.ui.Section;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.pdp.businessobject.inquiry.PaymentDetailInquirable;
+import org.kuali.rice.location.api.country.Country;
+
+import edu.cornell.kfs.pdp.CUPdpPropertyConstants;
+
+@SuppressWarnings("deprecation")
+public class CuPaymentDetailInquirable extends PaymentDetailInquirable {
+    private static final long serialVersionUID = -5852448786635585758L;
+
+    /**
+     * Overridden to prevent exceptions when the country field is blank,
+     * or when it contains a country name instead of a country code.
+     * 
+     * @see org.kuali.kfs.pdp.businessobject.inquiry.PaymentDetailInquirable#convertCountryForDisplay(org.kuali.kfs.kns.web.ui.Section)
+     */
+    @Override
+    protected void convertCountryForDisplay(Section section) {
+        section.getRows().stream()
+                .flatMap((row) -> row.getFields().stream())
+                .filter(this::isPaymentGroupCountryField)
+                .findFirst()

```

Comment:
do we only need to update the display for the first field, and not the rest?

Reply:
There should theoretically be only one field with the "paymentGroup.country" property name. The superclass's version of this method also short-circuits after updating the first match, so I was trying to be consistent with the superclass's logic as much as possible.

Reply:
OK, lets go with as is then